portfolio/src/main/webapp/blog.html

```diff
@@ -53,11 +54,22 @@ <h4>Software Engineering Intern</h4>
     <div class="menu">
       <section class="banner">
         <div class="info">
-          <div class="title"><h2>Blog Title</h2></div>
-          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. </p>
-          <button id="comment-btn">Get Comment!</button>
+          <div class ="title"><h2>Blog Title</h2></div>
+          
+          <p> Insert some blog post content here. </p>
+
+          <div class ="title"><h2>All Comments</h2></div>
           <div id="comment-container"></div>
 
+          <div class ="title"><h2>Leave A Comment</h2></div>
+          <form action="/data" method="POST">
+            <label for="name">Name:</label><br>
+            <input type="text" id="name" name="name"> <br>
```

Comment:
These br elements are used for styling purposes, which is not the right use.

The problem you're facing is that label and textarea are inline elements and not block, so the following elements remains on the same line. One solution to this is to give them a display value of block.

Flexbox is also a potential solution.

Reply:
Removed br, will adjust input boxes in next PR